src/functionalTest/resources/features/F-209 - Apply NoC Decision/F-209 - Apply NoC Decision.feature

```diff
@@ -0,0 +1,198 @@
+@F-209
+Feature:
+  AS AN invoker of the ApplyNoCDecision API
+  I WANT to check if a NoCRequest has been manually approved
+  SO THAT I can apply the necessary steps to the case in relation to the action being taken
+
+  @S-209.1
+  Scenario: AC2, AC1B Happy path  Apply NoCDecision to remove and replace representation when a NoCRequest has been approved (Happy Path)
+    Given an appropriate test context as detailed in the test data source,
+    Given a user [Richard - with the ability to create a case for a particular jurisdiction within an organisation],
+    And a user [Dil - with a solicitor role for the same jurisdiction within a different organisation from Richard],
+    And a successful call [by Richard to create a case - C1 - on behalf of Mario as the applicant which is auto-assigned to Richard's organisation] as in [F-209_NoC_Case_Creation_By_Richard_With_Assigned_Org_Policies],
+    And a successful call [to remove CREATOR assigned CaseRole] as in [F-209_Delete_Creator_Role],
+    And a successful call [to assign users to the case] as in [F-209_assign_access_to_case],
+    And a wait time of [5] seconds [to allow for Logstash to index the case just created],
```

Comment:
Might be wrong but I thought all the Data Store ES endpoint calls had been taken out, so we might be able to remove this and every other wait in this feature file?

Reply:
After looking through more of this PR, think this comment can be ignored as the ES endpoint is being used for verification?